### PR TITLE
STLX-15824_add_verify_transaction_in_core_for_near

### PR DESCRIPTION
**Pull request by fmartinAltamirano**
Add verify transaction in core for Near

### CODE REVIEW

File: modules/bitgo/src/v2/coins/near.ts [hunk 1]
```diff
@@ -166,15 +170,14 @@ export class Near extends BaseCoin {
...
     const factory = accountLib.register(this.getChain(), accountLib.Near.TransactionBuilderFactory);
     let rebuiltTransaction: accountLib.BaseCoin.BaseTransaction;
-    const txRaw = params.txPrebuild.txHex;
+    const txHex = params.txPrebuild.txHex;
+    const rawTxBase64 = HEX_REGEX.test(txHex) ? Buffer.from(txHex, 'hex').toString('base64') : txHex;
 
     try {
-      const transactionBuilder = factory.from(txRaw);
+      const transactionBuilder = factory.from(rawTxBase64);
       rebuiltTransaction = await transactionBuilder.build();
```
Comment (ipelaez-simtlix) [inline modules/bitgo/src/v2/coins/near.ts:181]: add a comment 
// TODO(STLX-16118) Change parse responsability to the from() method in Account-lib

Comment (fmartinAltamirano) [reply modules/bitgo/src/v2/coins/near.ts]: Done

File: modules/bitgo/src/v2/coins/near.ts [hunk 2]
```diff
@@ -166,15 +170,15 @@ export class Near extends BaseCoin {
...
-    params: ExplainTransactionOptions,
-  ): Promise<NearTransactionExplanation> {
+  async explainTransaction(params: ExplainTransactionOptions): Promise<NearTransactionExplanation> {
     const factory = accountLib.register(this.getChain(), accountLib.Near.TransactionBuilderFactory);
     let rebuiltTransaction: accountLib.BaseCoin.BaseTransaction;
-    const txRaw = params.txPrebuild.txHex;
+    const txHex = params.txPrebuild.txHex;
+    const rawTxBase64 = HEX_REGEX.test(txHex) ? Buffer.from(txHex, 'hex').toString('base64') : txHex;
 
+    // TODO(STLX-16118) Change parse responsability to the from() method in Account-lib
```
Comment (alebusse) [inline modules/bitgo/src/v2/coins/near.ts:179]: You can remove this, its already merge

File: modules/bitgo/src/v2/coins/near.ts [hunk 3]
```diff
@@ -263,22 +262,38 @@ export class Near extends BaseCoin {
...
       throw new Error('missing required tx prebuild property txHex');
     }
 
-    transaction.fromRawTransaction(rawTx);
-
-    // TO-DO: new explainTransaction to be implemented in account-lib
+    let rawTxBase64 = rawTx;
+    if (HEX_REGEX.test(rawTx)) {
+      rawTxBase64 = Buffer.from(rawTx, 'hex').toString('base64');
+    }
```
Comment (alebusse) [inline modules/bitgo/src/v2/coins/near.ts:277]: You can remove this, its already fixed

File: modules/bitgo/src/v2/coins/near.ts (hunk 2 above)
Comment (fmartinAltamirano) [reply modules/bitgo/src/v2/coins/near.ts]: Done

File: modules/account-lib/src/coin/near/transaction.ts [hunk 4]
```diff
@@ -62,7 +62,7 @@ export class Transaction extends BaseTransaction {
       parsedAction = {
         functionCall: {
           methodName: functionCallObject.methodName,
-          args: JSON.parse(functionCallObject.args.toString()),
+          args: JSON.parse(Buffer.from(functionCallObject.args).toString()),
```
Comment (alebusse) [inline modules/account-lib/src/coin/near/transaction.ts:65]: Whats this for?

Comment (ipelaez-simtlix) [reply modules/account-lib/src/coin/near/transaction.ts:65]: Sometimes buffer can come as a buffer or a string buffer, we could try to modify every case where args need to be parsed or just use Buffer.from() here which accept both types

**Review by rliberal-simtlix — APPROVED** [review]

**Review by ewangbitgo — APPROVED** [review]

**Pull request merged**